include/odp/api/spec/packet_io_stats.h

```diff
@@ -102,6 +103,48 @@ typedef struct odp_pktio_stats_t {
 	uint64_t out_errors;
 } odp_pktio_stats_t;
 
+/**
+ * Packet IO input queue specific statistics counters
+ *
+ * Statistics counters for an individual packet input queue. Refer to packet IO
+ * level statistics odp_pktio_stats_t for counter definitions.
+ */
+typedef struct odp_pktin_queue_stats_t {
+	/** @see odp_pktio_stats_t::in_octets */
+	uint64_t in_octets;
```

Comment:
Counters are already in the input (queue) namespace, so, I think, no reason to add "in_" prefix. The same applies to output queue.

Reply:
The original reasoning was to make the naming consistent with `odp_pktio_stats_t`, but now when we'll likely have also `odp_cls_queue_stats_t ` and `odp_tm_queue_stats_t ` stats without the prefixes, I'd tend to agree with you.

Reply:
Fixed in V4.